### ISSUE TITLE
An error from 2016

### ISSUE THREAD

**Issue by toplisek** [wummel/linkchecker#741]
I'm checking if it is solved an error from 2016:
Error: SSLError: [Errno 1] _ssl.c:510: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure

Is this outdated topic or solved?
https://www.ssllabs.com is A grade
Need help

**Comment by petrvecera** [comment]
Yep, let's encrypt SSL doesn't work too https://helloworld.letsencrypt.org/
```
Error: SSLError: [Errno 1] _ssl.c:510: error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed
```

**Comment by dpalic** [comment]
Thank you for the issue report. Sadly this project is dead, and a new team is around with https://github.com/linkcheck/linkchecker
for more details please see: #708
Also please close this issue and report it freshly on the new repo https://github.com/linkcheck/linkchecker/issues

**Comment by petrvecera** [comment]
The new repo sucks too https://github.com/linkcheck/linkchecker/issues/53

**Comment by dpalic** [comment]
you know it is a community. Nowadays it is pretty easy to make something which sucks into something great.
But all it depends: how do *YOU* support to make it great?
e.g. go on an create a doc how to build a binary and post it on the project wiki. 
=> your problem solved
=> you have given something back to the universe which gave you already such a great tool

(I am neither a developer nor a contributor, but I support by helping guys like you to find the right project. My way to give something back...)
